opentelemetry/CHANGELOG.md

```diff
@@ -104,10 +105,10 @@ let counter = meter.u64_counter("my_counter").build();
   - Replaced `global::meter_with_version` with `global::meter_with_scope`
   - Added `global::tracer_with_scope`
   - Refer to PR description for migration guide.
-- **Breaking change**: replaced `InstrumentationScope` public attributes by getters [#2275](https://github.com/open-telemetry/opentelemetry-rust/pull/2275)  
```

Comment:
Sorry for the whitespace stripped in this file by my editor, let me know if this needs reverting.